src/components/Questionnaire/QuestionnaireEditor.tsx

```diff
@@ -963,23 +998,23 @@ export default function QuestionnaireEditor({ id }: QuestionnaireEditorProps) {
                   </div>
 
                   <div>
-                    <Label htmlFor="slug">Slug</Label>
+                    <Label htmlFor="slug">{t("slug")}</Label>
                     <Input
                       id="slug"
                       value={questionnaire.slug}
                       onChange={(e) =>
                         updateQuestionnaireField("slug", e.target.value)
                       }
-                      placeholder="unique-identifier-for-questionnaire"
+                      placeholder={t("unique_identifier_question")}
                       className="font-mono"
                     />
                     <p className="text-sm text-gray-500 mt-1">
-                      A unique URL-friendly identifier for this questionnaire
+                      {t("a_unique_url_friendly_identifier_questionnaire")}
```

Comment:
for stuff like this, you don't need to write the whole thing as a key 😅 

`unique_url_id_questionnaire` would have worked.